cedar-policy-formatter/src/pprint/doc.rs

```diff
@@ -412,8 +412,8 @@ impl Doc for ASTNode<Option<RecInit>> {
             key_doc
                 .append(RcDoc::line_())
                 .append(add_comment(
-                    RcDoc::text(":"),
-                    Comment::default(),
+                    RcDoc::as_string(":"),
```

Comment:
Should we add a test case for this?

Reply:
Good point. I think I will add more tests in another PR since there aren't many tests for the formatter.